### PR TITLE
Remove deprecated order.TargetLocation

### PR DESCRIPTION
**Pull request by pchote**

### CODE REVIEW

File: OpenRA.Mods.Cnc/Traits/Minelayer.cs [hunk 1]
```diff
@@ -89,13 +89,17 @@ Order IIssueDeployOrder.IssueDeployOrder(Actor self, bool queued)
...
 		{
+			if (order.OrderString != "BeginMinefield" && order.OrderString != "PlaceMinefield" && order.OrderString != "PlaceMine")
+				return;
+
+			var cell = self.World.Map.CellContaining(order.Target.CenterPosition);
 			if (order.OrderString == "BeginMinefield")
-				minefieldStart = order.TargetLocation;
+				minefieldStart = cell;
 
 			if (order.OrderString == "PlaceMine")
```
Comment (reaperrr) [inline OpenRA.Mods.Cnc/Traits/Minelayer.cs]: Minor scope-creep: You could make this and the "PlaceMinefield" check below an `else if` while you're at it.

**Comment by reaperrr** [comment]
Also needs a rebase, 👍 after that.

**Comment by pchote** [comment]
Rebased and updated.

**Review by reaperrr — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d80ccaae0f6a6a069359502f4627859e1841767e

**Pull request merged**